### PR TITLE
Rebalance IG Boni

### PR DESCRIPTION
**Pull request by Tziplays**
Mexico:
add normal Army IG buffs on top of authority

Bolivia:
add normal Army IG buffs on top of authority

Brasil:
add Tech buff to inteligentsia IG
add Tech buff to Industrialists IG

France
change buff to 15% off 5% Def (30% Off 10% Def at powerful)

Persia:
add 5% EA to the one with 50% conversion

Egypt/Ottomans:
add 5% EA to the one with 50% conversion

Sweden:
industrialists IG
add 2.5% manufacturing income to the idea with mining income

Japan:
Church ig
change -50% -> -15% radicals from Sol decrease
add +1% pop growth

industrialists:
add 5% constr eff to zaibatsu

Prussia:
industrialist IG:
add 2.5% manufacturing TP to second passive
rename passive to Made in Germany

### CODE REVIEW

File: src/common/interest_group_traits/kmpmp_00_devout_traits.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿ig_trait_asceticism = {
+	icon = "gfx/interface/icons/ig_trait_icons/be_fruitful_and_multiply.dds"
+	min_approval = loyal
+	
+	modifier = {
+		state_radicals_from_sol_change_mult = -0.15 #0.5
```
Comment (MasterSon-git) [inline src/common/interest_group_traits/kmpmp_00_devout_traits.txt:6]: should be positive

File: src/common/interest_group_traits/kmpmp_00_devout_traits.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		state_accepted_birth_rate_mult = 0.01
```
Comment (MasterSon-git) [inline src/common/interest_group_traits/kmpmp_00_devout_traits.txt:7]: not only accepted pops

File: src/common/interest_groups/kmpmp_00_industrialists.txt [hunk 3]
```diff
@@ -0,0 +1,363 @@
+﻿	ig_industrialists = {
```
Comment (MasterSon-git) [inline src/common/interest_groups/kmpmp_00_industrialists.txt:1]: indents

File: src/localization/english/kmpmp_l_english.yml [hunk 4]
```diff
@@ -169,3 +169,4 @@
  kmpmp_ev_persia.1.a:0 "Hooray"
 
  kmpmp_china_buff:0 "China buff"
+ kmpmp_ig_trait_made_in_germany:0 "Deutsche Qualität?"
```
Comment (MasterSon-git) [inline src/localization/english/kmpmp_l_english.yml:172]: why not "Made in Germany"?

**Review by MasterSon-git — CHANGES_REQUESTED** [review]

File: src/common/interest_group_traits/kmpmp_00_devout_traits.txt (hunk 1 above)
Comment (Tziplays) [reply src/common/interest_group_traits/kmpmp_00_devout_traits.txt]: ist gemacht

**Review by MasterSon-git — APPROVED** [review]

**Pull request merged**